### ISSUE TITLE
[Accessibility] Vignette plots lack alt= attributes

### ISSUE THREAD

**Issue by MichaelChirico** [thomasp85/ggraph#340]
We have a tool that flags images that lack an `alt=` attribute. See https://help.siteimprove.com/support/solutions/articles/80000863904-accessibility-image-alt-text-best-practices -- these are essential for conveying the image contents to users on screen readers.

You can add these to your vignettes by using the recent-ish `fig.alt` chunk option in `knitr`; see https://www.rstudio.com/blog/knitr-fig-alt/.

There are a lot of issues, so I'll just name the vignettes:

 * Edges
 * Layouts
 * Nodes
 * tidygraph

If you'd like more details I can provide, but I'm assuming it's basically clear enough what plots are at issue.
